### ISSUE TITLE
Add Example of Multiple Writes in a Single Transaction to 'How to make multiple writes in a transaction' Tutorial

### ISSUE THREAD

**Issue by romainwg** [etcd-io/website#901]
**Current Title of the Web Page:** [How to make multiple writes in a transaction](https://etcd.io/docs/v3.5/tutorials/how-to-transactional-write/)

**Feedback:**

I noticed that the current example on the page primarily demonstrates the structure of transactions using `if`, `then`, `else`, and `commit` statements. However, it does not include an example of making multiple writes within a single transaction.

To enhance the technical clarity and usefulness of this tutorial, I suggest adding an example (or modifying the existing one) to explicitly show how to perform multiple writes in a single transaction. Demonstrating a batch of operations would be particularly beneficial for users looking to understand this functionality better. For instance, an example that includes multiple `Put` operations within a single transaction would be very helpful.

Thank you for considering this improvement.

**Comment by jmhbnz** [comment]
Thanks for raising this @romainwg - I agree that page could certainly have some more comprehensive examples.

We should also replicate the change across to the 3.6 version page https://etcd.io/docs/v3.6/tutorials/how-to-transactional-write.

If you have time to propose a pull request please give it a try. I am happy to answer any questions and provide feedback 🙏🏻

**Comment by NavinShrinivas** [comment]
I'll give this a go! I will make a pr today or tmrw.

**Comment by NavinShrinivas** [comment]
https://github.com/user-attachments/assets/b599e16e-b1a8-401f-a4e4-262f294f0e5e

I have this gif showing two examples, one with updating/writing to multiple keys 

and the second trying two writes to the same transaction and failing

**Comment by NavinShrinivas** [comment]
Noted! Did the changes in the pr. handled lint too. 

awaiting your feedback :)

**Comment by romainwg** [comment]
> Thanks for raising this @romainwg - I agree that page could certainly have some more comprehensive examples.
> 
> We should also replicate the change across to the 3.6 version page https://etcd.io/docs/v3.6/tutorials/how-to-transactional-write.
> 
> If you have time to propose a pull request please give it a try. I am happy to answer any questions and provide feedback 🙏🏻

Sorry for the delay, here's what I did: https://github.com/etcd-io/website/pull/908

**Comment by ivanvc** [comment]
Hi @romainwg and @NavinShrinivas. I reviewed both of your pull requests. I wouldn't say that one is correct (or better than the other one). But it definitely adds contention to resolve this issue. Given that technically this wasn't assigned to anybody, and that @romainwg originally opened the issue, I would suggest to:

1. Work together and co-author the commits in a single pull request (ref: [GitHub documentation](https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors)).
2. We work on the pull requests in a chronological order, and if there are any improvements to @NavinShrinivas, @romainwg works on top of it.

Ideally, I would prefer a single pull request (1). But let me know your thoughts.

**Comment by romainwg** [comment]
Thank you @ivanvc 

I don't want to sound pretentious, but I think our proposals are orthogonal.

On #908:

* I've aligned the tutorial for `transactions` with the one for `watch` & `delete`.
* I've also added a "real world" example which should be more meaningful

(And it may be too big for just one PR, but it's simpler for me.)

Also, when I posted the issue I had already started thinking about the best way to illustrate the transaction documentation.
Due to lack of time, I wasn't able to propose it straight away.

---

If it makes things easier for the maintainers, I'll commit to the first PR, but my changes are quite different...
